package.json

```diff
@@ -146,7 +145,9 @@
     "moduleNameMapper": {
       "^@/(.*)$": "<rootDir>/src/$1",
       "^Components/(.*)$": "<rootDir>/src/components/$1",
-      "^.+\\.(css|less|svg)$": "<rootDir>/tests/javascript/helpers/CSSStub.js"
+      "^.+\\.(css|less|svg)$": "<rootDir>/tests/javascript/helpers/CSSStub.js",
+      "^unist-util-visit$": "<rootDir>/tests/javascript/helpers/unist-stub.js",
```

Comment:
this seems like the easiest solution to me, good fix!

Reply:
> this seems like the easiest solution to me, good fix!

@devlinjunker thanks for the review